src/components/dashboard/burn_tokens.test.tsx

```diff
@@ -0,0 +1,119 @@
+
+				}} parentSetters={{                                
+					setNewIbcIssuance: setNewIbcIssuance,
+					setNewPrice: setNewPrice,
+					setNewReserve: setNewReserve}} wallet={null}/>
+			</ChakraProvider>
+		)
+		const inputElement = getByTestId('you_pay')
+
+		fireEvent.change(inputElement, {target:{value:'11'}})
+
+		await waitFor(() => {
+			expect(newPrice).toBe('1571890826397969483')
+			expect(newIbcIssuance.toString()).toBe(ethers.BigNumber.from('14095447126455587752').sub(ethers.BigNumber.from('10889999999999999999')).toString())
+			expect(newReserve).toBe(ethers.BigNumber.from('21131781542148949646').sub(ethers.BigNumber.from('11054555675653151020')).toString())
+		});
+	});
+});
+
+
+describe('BurnToken YOU RECEIVE handling', () => {
+	it('', async () => {
+
+		let newPrice: any;
+		function setNewPrice(x:any){
+			newPrice = x
+		}
+
+		let newReserve: any;
+		function setNewReserve(x:any){
+			newReserve = x
+		}
+
+		let newIbcIssuance: ethers.BigNumber = ethers.BigNumber.from('0');
+		function setNewIbcIssuance(x:ethers.BigNumber){
+			newIbcIssuance = x
+		}
+
+		const { getByTestId } = render(
+			<ChakraProvider theme={theme}>
+				<CSSReset />
+				<BurnTokens dashboardDataSet={{
+					bondingCurveParams: {
+						reserveAmount: "21131781542148949646",
+						inverseTokenSupply: "14095447126455587752",
+						invariant: "5628552015782940183",
+						currentTokenPrice: "749595999068626357"
+					},
+					reserveTokenDecimals: ethers.BigNumber.from("18"),
+					inverseTokenDecimals: ethers.BigNumber.from("18"),
+					fees: {"lpFee":{"buyTokens":"2500000000000000","sellTokens":"2500000000000000","addLiquidity":"2500000000000000","removeLiquidity":"2500000000000000"},"stakingFee":{"buyTokens":"2500000000000000","sellTokens":"2500000000000000","addLiquidity":"2500000000000000","removeLiquidity":"2500000000000000"},"protocolFee":{"buyTokens":"5000000000000000","sellTokens":"5000000000000000","addLiquidity":"5000000000000000","removeLiquidity":"5000000000000000"}}
+				}} parentSetters={{                                
+					setNewIbcIssuance: setNewIbcIssuance,
+					setNewPrice: setNewPrice,
+					setNewReserve: setNewReserve}} wallet={null}/>
+			</ChakraProvider>
+		)
+		const inputElement = getByTestId('you_receive')
+
+		fireEvent.change(inputElement, {target:{value:'10.944'}})
+
+		await waitFor(() => {
+			expect(newPrice).toBe('1571889231955141735')
+			expect(newIbcIssuance.toString()).toBe('3205453629956167694')
+			expect(newReserve).toBe(ethers.BigNumber.from('21131781542148949646').sub(ethers.BigNumber.from('11054545454545455629')).toString())
+		});
+	});
+});
+
+/*
+2814276007789739754535245660
```

Comment:
what's the purpose of this commented out numbers? 0_0??

Reply:
fixed in https://github.com/bear-market-labs/ibc_interface/pull/252/commits/2ae49c487912629ec7451edee1619e03bf903aee